docs/source/p_soc_auto/uwsgi.rst

```diff
@@ -43,3 +43,8 @@ at the same time
 
     [root@lvmsocq01 ~]# systemctl restart nginx;systemctl restart uwsgi-phsa-soc-app.service
     [root@lvmsocq01 ~]#
+
+.. note::
+
+    We are currently using a custom uwsgi executable. When setting up a
+    new system copy the uwsgi file located on lvmsockq01 in /usr/bin.
```

Comment:
it's lvmsocqo1, not lvmsockqo1. and there is a valid uwsgi executable on lvmsocq02.
Please add a todo here to resolve this thing cleanly (and matching trello story on the IOUs board.